utam-core/src/test/java/utam/core/selenium/element/ElementAdapterTests.java

```diff
@@ -53,32 +55,84 @@
  */
 public class ElementAdapterTests {
 
+  final static String NOT_FOUND_SELECTOR = "not_found";
+  final static String ELEMENT_NOT_FOUND_ERROR = "can't find element with locator 'By.cssSelector: not_found'";
+
+  static ElementAdapter findNotNullable(LocatorBy locator, Element scope) {
+    return  (ElementAdapter) scope.findElement(locator, false);
+  }
+
+  static ElementAdapter findNullable(LocatorBy locator, Element scope) {
+    Element res = scope.findElement(locator, true);
+    return res == null? null : (ElementAdapter) res;
+  }
+
+  static List<Element> findNotNullables(LocatorBy locator, Element scope) {
+    return  scope.findElements(locator, false);
+  }
+
+  static List<Element> findNullables(LocatorBy locator, Element scope) {
+    return scope.findElements(locator, true);
+  }
+
   @Test
-  public void testFind() {
+  public void testFindElement() {
     MockUtilities mock = new MockUtilities();
     WebElement foundMock = mock(WebElement.class);
-    Element test = mock.getElementAdapter();
-    when(mock.getWebElementMock().findElement(By.cssSelector("css")))
-        .thenReturn(foundMock);
-    ElementAdapter found = (ElementAdapter) test.findElement(byCss("css"));
+    Element scope = mock.getElementAdapter();
+    final String FOUND_SELECTOR = "found";
+    when(mock.getWebElementMock().findElements(By.cssSelector(FOUND_SELECTOR)))
+        .thenReturn(Collections.singletonList(foundMock));
+    ElementAdapter found = findNotNullable(byCss(FOUND_SELECTOR), scope);
     assertThat(found.getWebElement(), is(sameInstance(foundMock)));
-    assertThrows(() -> test.findElement(byCss("css1")));
-    when(mock.getWebElementMock().findElements(By.cssSelector("css")))
-        .thenReturn(Collections.singletonList(mock(WebElement.class)));
-    assertThat(test.findElements(byCss("css")),
-        is(not(empty())));
-    assertThrows(() -> test.findElements(byCss("css1")));
   }
 
   @Test
-  public void testFindInShadow() {
+  public void testFindElementNotFound() {
+    Element scope = new MockUtilities().getElementAdapter();
+    Exception e = expectThrows(NoSuchElementException.class, () -> findNotNullable(byCss(NOT_FOUND_SELECTOR), scope));
+    assertThat(e.getMessage(), containsString(ELEMENT_NOT_FOUND_ERROR));
+  }
+
+  @Test
+  public void testFindElementNullable() {
+    MockUtilities mock = new MockUtilities();
+    WebElement foundMock = mock(WebElement.class);
+    Element scope = mock.getElementAdapter();
+    final String FOUND_SELECTOR = "found";
+    when(mock.getWebElementMock().findElements(By.cssSelector(FOUND_SELECTOR)))
+        .thenReturn(Collections.singletonList(foundMock));
+    ElementAdapter found = findNullable(byCss(FOUND_SELECTOR), scope);
+    assertThat(found.getWebElement(), is(sameInstance(foundMock)));
+  }
+
+  @Test
+  public void testFindElementNullableNotFound() {
+    Element scope = new MockUtilities().getElementAdapter();
+    assertNull(findNullable(byCss(NOT_FOUND_SELECTOR), scope));
```

Comment:
Any reason we use `assertNull` here instead of the `assertThat(findNullable(byCss(NOT_FOUND_SELECTOR), scope), is(nullValue())`? Other than convenience? Makes it inconsistent with all of the other assert patterns in the unit tests.